### DISCUSSION TITLE
can not delete vela resouce

### DISCUSSION THREAD

**Discussion by zhuchubo** [kubevela/kubevela#2919] [Q&A]
$ kubectl get crd |grep oam
applications.core.oam.dev                                2021-10-12T03:37:38Z

 kubectl get application -A
NAMESPACE     NAME               COMPONENT                               TYPE   PHASE     HEALTHY   STATUS   AGE
vela-system   fluxcd             alerts.notification.toolkit.fluxcd.io   raw    running   true               62d
vela-system   ns-flux-system     flux-system                             raw    running   true               62d
vela-system   vela-env-default                                                  running                      62d

even if kubectl delete application --force --grace-time 0 and use kubernetes finalizers. Can not delete crd applications.core.oam.dev  and application.
help !

**Comment by Somefive** [comment]
You can check the status of the application you want to deleted. It should enter `deleting` phase. If something blocks the deletion, it could be some resources not recycled correctly such as HelmRelease.
